src/Spice86.Core/Emulator/OperatingSystem/MountedFolder.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Spice86.Core.Emulator.OperatingSystem;
+
+/// <summary>
+/// Represents a host folder used as a drive by DOS.
+/// </summary>
+public class MountedFolder {
+    /// <summary>
+    /// Initializes a new instance.
+    /// </summary>
+    /// <param name="driveLetter">The DOS driver letter.</param>
+    /// <param name="fullName">The full host path to the folder to be used as the DOS drive root.</param>
+    public MountedFolder(char driveLetter, string fullName) {
+        DriveLetter = driveLetter;
+        MountPoint = fullName;
+        CurrentDirectory = "";
+    }
+
+    /// <summary>
+    /// Gets the DOS drive letter.
+    /// </summary>
+    public char DriveLetter { get; init; }
+
+    /// <summary>
+    /// The full host path to the mounted folder. This path serves as the root of the DOS drive.
+    /// </summary>
+    public string MountPoint { get; init; }
```

Comment:
Would be clearer if every path had a host / dos prefix to get better what it is at glance, especially when it is used

Reply:
Done.